### PR TITLE
[Localization] Translate summary

### PR DESCRIPTION
**Pull request by Nolway**
## What are the changes?
Translate missing sentences and words on the pokemon summary.

## Why am I doing these changes?
Because few sentences and words aren't translated.

## What did change?
Theses sentences will be translatable.

### Screenshots/Videos
image
image
image

## How to test the changes?
Go into a Pokemon summary with the french language.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/pokemon-summary.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const pokemonSummary: SimpleTranslationEntries = {
+  "pokemonInfo": "Info Pokémon",
+  "status": "Status",
```
Comment (Adri1) [inline src/locales/fr/pokemon-summary.ts:5]:
```suggestion
  "status": "Statut",
```

**Review by Adri1 — COMMENTED** [review]
Please change theses French entries

File: src/locales/zh_CN/pokemon-summary.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "type": "Type",
+  "unknownTrainer": "Unknown",
+  "ot": "OT",
+  "luck": "Luck",
+  "nature": "nature",
+  "apparently": "apparently",
+  "metAtLv": "met at Lv.",
+  "expPoints": "Exp. Points",
+  "nextLv": "Next Lv.",
+  "cancel": "Cancel",
```
Comment (40chyan) [inline src/locales/zh_CN/pokemon-summary.ts:16]:
```suggestion
"pokemonInfo": "宝可梦信息",
  "status": "状态",
  "powerAccuracyCategory": "威力\n命中率\n类别",
  "type": "属性",
  "unknownTrainer": "未知",
  "ot": "训练师",
  "luck": "幸运",
  "nature": "性格",
  "apparently": "命中注定般地",
  "metAtLv": "相遇于Lv.",
  "expPoints": "经验值",
  "nextLv": "下一级",
  "cancel": "取消",
```

File: src/locales/ko/pokemon-summary.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "type": "Type",
+  "unknownTrainer": "Unknown",
+  "ot": "OT",
+  "luck": "Luck",
+  "nature": "nature",
+  "apparently": "apparently",
+  "metAtLv": "met at Lv.",
+  "expPoints": "Exp. Points",
+  "nextLv": "Next Lv.",
+  "cancel": "Cancel",
```
Comment (returntoice) [inline src/locales/ko/pokemon-summary.ts:16]:
```suggestion
  "pokemonInfo": "포켓몬 정보",
  "status": "스테이터스",
  "powerAccuracyCategory": "위력\n명중\n분류",
  "type": "타입",
  "unknownTrainer": "Unknown",
  "ot": "어버이",
  "luck": "럭",
  "nature": "성격",
  "apparently": "분명히",
  "metAtLv": "만난 레벨은 Lv.",
  "expPoints": "현재 경험치",
  "nextLv": "다음 레벨까지",
  "cancel": "돌아간다",
```

**Review by returntoice — CHANGES_REQUESTED** [review]

File: src/locales/ko/pokemon-summary.ts (hunk 3 above)
Comment (ddullgi) [inline src/locales/ko/pokemon-summary.ts:16]:
```suggestion
  "pokemonInfo": "포켓몬 정보",
  "status": "능력치",
  "powerAccuracyCategory": "위력\n명중\n분류",
  "type": "타입",
  "unknownTrainer": "알수없음",
  "ot": "어버이",
  "luck": "행운",
  "nature": "성격",
  "apparently": "apparently",
  "metAtLv": "met at Lv.",
  "expPoints": "현재 경험치",
  "nextLv": "다음 레벨까지",
  "cancel": "그만둔다",
```

Due to differences in word order between Korean and English, those parts have not been translated.

**Review by ddullgi — CHANGES_REQUESTED** [review]

File: src/locales/ko/pokemon-summary.ts (hunk 3 above)
Comment (returntoice) [reply src/locales/ko/pokemon-summary.ts:16]: "status"는 9세대 스칼렛/바이올렛 기준으로 "스테이터스"라고 되어 있어 "능력치"보다는 "스테이터스"가 좋을 듯합니다. 마찬가지로 "cancel"도 "돌아간다"로 되어 있습니다.

Comment (ddullgi) [reply src/locales/ko/pokemon-summary.ts:16]: 먼저 올리신 걸 나중에 봤네요.
의견주신 것들 고치면 거의 일치하니까 
제안은 취소할께요

File: src/locales/de/pokemon-summary.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  "unknownTrainer": "Unknown",
+  "ot": "OT",
+  "luck": "Luck",
+  "nature": "nature",
+  "apparently": "apparently",
+  "metAtLv": "met at Lv.",
+  "expPoints": "Exp. Points",
+  "nextLv": "Next Lv.",
+  "cancel": "Cancel",
+} as const;
```
Comment (CodeTappert) [inline src/locales/de/pokemon-summary.ts:17]:
```suggestion
export const pokemonSummary: SimpleTranslationEntries = {
  "pokemonInfo": "Pokémon Info",
  "status": "Status",
  "powerAccuracyCategory": "Stärke\nGenauigkeit\nKategorie",
  "type": "Typ",
  "unknownTrainer": "Unbekannt",
  "ot": "OT",
  "luck": "Glück",
  "nature": "Wesen",
  "apparently": "Wahrscheinlich",
  "metAtLv": "getroffen auf Lvl. ",
  "expPoints": "Erf. Punkte",
  "nextLv": "Nächstes Lvl.",
  "cancel": "Abbrechen",
} as const;
```

File: src/locales/fr/pokemon-summary.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  "powerAccuracyCategory": "Puissance\nPrécision\nCatégorie",
+  "type": "Type",
+  "unknownTrainer": "Inconnu",
+  "ot": "DO",
```
Comment (Dakurei) [inline src/locales/fr/pokemon-summary.ts:9]:
```suggestion
  "ot": "D.O.",
```
image

File: src/locales/de/pokemon-summary.ts (hunk 4 above)
Comment (CodeTappert) [reply src/locales/de/pokemon-summary.ts:17]: Please test if this actually fits

**Comment by NicusPulcis** [comment]
Italian translations have been sent

File: src/locales/fr/config.ts [hunk 6]
```diff
@@ -39,7 +39,11 @@ import { tutorial } from "./tutorial";
 import { voucher } from "./voucher";
 import { weather } from "./weather";
 import { partyUiHandler } from "./party-ui-handler";
+<<<<<<< HEAD
 import { settings } from "#app/locales/fr/settings.js";
+=======
+import { pokemonSummary } from "./pokemon-summary";
+>>>>>>> Translate pokemon summary to fr
```
Comment (Dakurei) [inline src/locales/fr/config.ts:46]: You commit a merge conflict

File: src/locales/es/pokemon-summary.ts [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  "type": "Type",
+  "unknownTrainer": "Unknown",
+  "ot": "OT",
+  "luck": "Luck",
+  "nature": "nature",
+  "apparently": "apparently",
+  "metAtLv": "met at Lv.",
+  "expPoints": "Exp. Points",
+  "nextLv": "Next Lv.",
+  "cancel": "Cancel",
```
Comment (GoldTra) [inline src/locales/es/pokemon-summary.ts:16]:
```suggestion
  "pokemonInfo": "Info. Pokémon",
  "status": "Estado",
  "powerAccuracyCategory": "Potencia\nPrecisión\nCategoria",
  "type": "Tipo",
  "unknownTrainer": "Desconocido",
  "ot": "EO",
  "luck": "Suerte",
  "nature": "de naturaleza",
  "apparently": "aparentemente",
  "metAtLv": "encontrado al Nv.",
  "expPoints": "Puntos Exp.",
  "nextLv": "Nv. siguiente",
  "cancel": "Salir",
```

Comment (GoldTra) [reply src/locales/es/pokemon-summary.ts:16]: Spanish Translations

Comment (GoldTra) [inline src/locales/es/pokemon-summary.ts:16]:
```suggestion
  "pokemonInfo": "Info. Pokémon",
  "status": "Estado",
  "powerAccuracyCategory": "Potencia\nPrecisión\nCategoria",
  "type": "Tipo",
  "unknownTrainer": "Desconocido",
  "ot": "EO",
  "luck": "Suerte",
  "nature": "de naturaleza",
  "apparently": "aparentemente",
  "metAtLv": "encontrado al Nv.",
  "expPoints": "Puntos Exp.",
  "nextLv": "Nv. siguiente",
  "cancel": "Salir",
```

**Review by GoldTra — COMMENTED** [review]
Spanish translations

**Comment by MrWaterT** [comment]
Can you check PR[#5](https://github.com/Nolway/pokerogue/pull/5), [#6](https://github.com/Nolway/pokerogue/pull/6) for text modification in some language?

**Comment by Nolway** [comment]
> Can you check PR[#5](https://github.com/Nolway/pokerogue/pull/5), [#6](https://github.com/Nolway/pokerogue/pull/6) for text modification in some language?

I merged the 6 and I'll wait for the 5 =)

File: src/locales/ko/pokemon-summary.ts [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales.js";
+
+export const pokemonSummary: SimpleTranslationEntries = {
+  "pokemonInfo": "포켓몬 정보",
+  "status": "능력치",
```
Comment (returntoice) [inline src/locales/ko/pokemon-summary.ts:5]:
```suggestion
  "status": "스테이터스",
```
The current version(9th gen SV) officially translates "status" into "스테이터스"

**Review by returntoice — CHANGES_REQUESTED** [review]

**Comment by MrWaterT** [comment]
I corrected both suggestion about "metAtLv" and "status" in [#5](https://github.com/Nolway/pokerogue/pull/5).
It is ready to be merged now.

File: src/locales/pt_BR/pokemon-summary.ts [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+  "type": "Type",
+  "unknownTrainer": "Unknown",
+  "ot": "OT",
+  "luck": "Luck",
+  "nature": "nature",
+  "apparently": "apparently",
+  "metAtLv": "met at Lv.",
+  "expPoints": "Exp. Points",
+  "nextLv": "Next Lv.",
+  "cancel": "Cancel",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/pokemon-summary.ts:16]:
```suggestion
  "pokemonInfo": "Informações do Pokémon",
  "status": "Status",
  "powerAccuracyCategory": "Poder\nPrecisão\nCategoria",
  "type": "Tipo",
  "unknownTrainer": "Desconhecido",
  "ot": "TO",
  "luck": "Sorte",
  "nature": "natureza",
  "apparently": "aparentemente",
  "metAtLv": "encontrado no Nv.",
  "expPoints": "Pontos EXP.",
  "nextLv": "Próx. Nv.",
  "cancel": "Cancelar",
```

**Comment by EnochG1** [comment]
@Nolway,
Could you merge branch 'main' into your branch?
I want to continue this PR.
Or if it's difficult for you to continue this PR on your own, I would mirror yours and continue.

**Review by returntoice — APPROVED** [review]
okay for Korean trans

**Comment by CodeTappert** [comment]
Please fix your conflicts

**Comment by NicusPulcis** [comment]
@CodeTappert sorry i can't find them, could you DM me on discord to help?

**Comment by EnochG1** [comment]
Mirror PR created at https://github.com/pagefaultgames/pokerogue/pull/2907

**Pull request closed without merging**